opentelemetry-contrib/Cargo.toml

```diff
@@ -34,7 +34,7 @@ serde_json = { version = "1", optional = true}
 futures = {version = "0.3", optional = true}
 async-trait = {version = "0.1", optional = true}
 tokio = { version = "1.0", features = ["fs", "io-util"], optional = true }
-async-std = { version = "1.6", optional = true }
+async-std = { version = "1.10", optional = true }
```

Comment:
Although it probably won't break the MSRV, we should probably pin this to 1.10 to be consistent with other crates?

Reply:
This was only to get CI passing. Still have to resolve the MSRV issues on main.